ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/SeedService.scala

```diff
@@ -6,9 +6,10 @@ package com.daml.ledger.participant.state.v1
 import java.security.SecureRandom
 
 import com.daml.lf.crypto
+import org.slf4j.LoggerFactory
```

Comment:
Can you use `ContextualizedLogger` and `LoggingContext` instead please?

Reply:
I cannot easily because it is the construction of a singleton object. 
Futhermore @stefanobaghino-da argued that:

Reply:
I don't know who that guy is, but he sounds smart, I'd probably follow his suggestion. 😛